Apps/W1/Shopify/app/src/Customers/Codeunits/ShpfyCustomerExport.Codeunit.al

```diff
@@ -87,9 +88,12 @@ codeunit 30116 "Shpfy Customer Export"
     var
         ShopifyCustomer: Record "Shpfy Customer";
         CustomerAddress: Record "Shpfy Customer Address";
+        EmptyEmailAddressLbl: Label 'Customer has no e-mail address.';
     begin
-        if Customer."E-Mail" = '' then
+        if Customer."E-Mail" = '' then begin
+            ShopifySkipRecordMgt.LogSkippedRecord(Customer.RecordId, EmptyEmailAddressLbl, Shop);
```

Comment:
This will log same record as skipped multiple times if user tries to sync multiple times. Should we do some duplicate handling? @AndreiPanko what do you think?